Analysis/Core/include/AnalysisCore/HFSelectorCuts.h

```diff
@@ -0,0 +1,86 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef HF_D0_SELECTOR_HELPERS_H_
+#define HF_D0_SELECTOR_HELPERS_H_
+#include <vector>
+#include <string>
+namespace o2::analysis
+{
+// namespace per channel?
+namespace HF_cuts_D0_topiK
+{
+static constexpr int npTBins = 25;
+static constexpr int nCutVars = 11 + 2;
+
+// not defined in ROOT's enum
+static constexpr int kD0 = 421;
+static constexpr int kD0bar = -421;
+
+// default values for the cuts
+constexpr double D0_cuts[npTBins][nCutVars] = {{0.0, 0.5, 0.400, 350. * 1E-4, 0.8, 0.5, 0.5, 1000. * 1E-4, 1000. * 1E-4, -5000. * 1E-8, 0.80, 0., 0.},      /* pt<0.5*/
+                                               {0.5, 1.0, 0.400, 350. * 1E-4, 0.8, 0.5, 0.5, 1000. * 1E-4, 1000. * 1E-4, -5000. * 1E-8, 0.80, 0., 0.},      /* 0.5<pt<1*/
```

Comment:
Why are there two pT columns now?

Reply:
These are bin limits, since we want to make them adjustable as well.

Reply:
I understand why the pT limits are in the array. It is not clear to me though why they cannot be represented by a single column of lower limits. It seems to me that a column with upper limits is needed only in case of discontinuous binning which I don't will ever be required.

Reply:
True. And apparently there is already an entry in my notes that I need to make it a single number... Thanks.

Reply:
Could there be a case of discontinuos binning? Also the last bin would need some upper limit cut

Reply:
So what is the conclusion? Should I use single number as a bin limit? Global upper and lower limits can be added as separate configurables.

Reply:
Personally i would stick with a lower and upper so as not to curtail any flexibility. But I have no strong opinions

Reply:
@nzardosh Unless there is a hint of why this flexibility would ever be needed in future, I would prefer safety now and go for the single column option.